client/src/components/TopBar.js

```diff
@@ -68,11 +68,13 @@ class BaseTopBar extends Component {
 
     render() {
         return (
-            <div style={{ flex:'0 0 auto'}}>
-                {this.props.currentUser && this.props.position && this.props.position.id === 0 && !this.props.isNewUser() && <NoPositionBanner />}
-                <GeneralBanner options={this.bannerOptions()} />
-                <SecurityBanner location={this.props.location} />
-                <Header minimalHeader={this.props.minimalHeader} toggleMenuAction={this.props.toggleMenuAction}/>
+            <div style={{ display: 'block', width: '100%', position: 'fixed'}}>
```

Comment:
A fixed layout breaks when the layout when the banner is bigger (i.e. when a banner announcement is on).
What is the "clicking add filters" problem being solved?

Reply:
Ah, is that overcome by https://github.com/NCI-Agency/anet/pull/651/commits/f98a544cb7b3ee8263750b93fb2da964959f3ca9 ?